src/modules/esl-utils/dom/events/target.ts

```diff
@@ -15,29 +19,42 @@ export class SyntheticEventTarget implements EventTarget {
       minCount = type;
       type = 'change';
     }
-    return this._listeners.size > minCount;
+    return this._listeners[type]?.length > minCount;
   }
 
   public addEventListener(callback: EventListenerOrEventListenerObject): void;
-  public addEventListener(type: 'change', callback: EventListenerOrEventListenerObject): void;
-  public addEventListener(type: any, callback: EventListenerOrEventListenerObject = type): void {
+  public addEventListener(type: string, callback: EventListenerOrEventListenerObject): void;
+  public addEventListener(type: string | EventListenerOrEventListenerObject = 'change', callback?: EventListenerOrEventListenerObject): void {
```

Comment:
```suggestion
  public addEventListener(type: string | EventListenerOrEventListenerObject, callback?: EventListenerOrEventListenerObject): void {
```
Method does not have empty argument definition